sdk/identity/identity/test/manual-integration/AzureVM/INSTRUCTIONS.md

```diff
@@ -0,0 +1,126 @@
+# prerequisite tools
+- Azure CLI
+
+# Azure resources
+This test requires instances of these Azure resources:
+- Azure Key Vault
+- Azure Managed Identity
+  - with secrets/set and secrets/delete permission for the Key Vault
+- Azure Virtual Machine with system-assigned identity
+- Azure Virtual Machine with user-assigned identity
+  - don't use the same VM twice
+
+The rest of this section is a walkthrough of deploying these resources.
+
+## Set environment variables to simplify copy-pasting
+- RESOURCE_GROUP
+  - name of an Azure resource group
+  - must be unique in the Azure subscription
+  - e.g. 'identity-test-rg'
+- VM_NAME_SYSTEM_ASSIGNED
+  - name of an Azure Virtual machine with a system-assigned identity
+  - must be unique in the resource group
+  - e.g. 'identity-test-vm-system'
+- VM_NAME_USER_ASSIGNED
+  - name of an Azure Virtual machine with a user-assigned identity
+  - must be unique in the resource group
+  - e.g. 'identity-test-vm-user'
+- MANAGED_IDENTITY_NAME
+  - name of the user-assigned identity
+  - 3-128 alphanumeric characters
+  - must be unique in the resource group
+- KEY_VAULT_NAME
+  - 3-24 alphanumeric characters
+  - must begin with a letter
+  - must be globally unique
+
+## Run inside of PowerShell
+
+These instructions assume you're running inside of PowerShell.
+
+## resource group
+```sh
+az group create -n $RESOURCE_GROUP --location westus2
```

Comment:
@danieljurek I believe you were mentioning to me earlier that you wanted to start working on some Azure CLI examples that could be useful for our ARM templates. Seems like this PR can be relevant to you!